### PR TITLE
Replace hard-coded strings with the translation function in the _dashboard.html.erb file and add new strings to the en.yml file.

### LINKED ISSUE #11074: Replace hard-coded strings with the translation function in the _dashboard.html.erb file and add new strings to the en.yml file.
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [app/views/sidebar/_dashboard.html.erb](https://github.com/publiclab/plots2/blob/main/app/views/sidebar/_dashboard.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -4,7 +4,7 @@
     <div class="btn-group" role="group">
       <a class="btn btn-primary btn-lg dropdown-toggle fa-white" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
         <span class="caret"></span>
-        <span class="sr-only">Toggle Dropdown</span>
+        <span class="sr-only"><%= translation('dashboard._header.toggle_dropdown') %></span>
       </a>

@@ -21,14 +21,14 @@
   <% if current_user.has_power_tag("lat") && current_user.has_power_tag("lon")%>
     <div style="margin: 2.2rem 0;">
       <a class="btn btn-lg btn-outline-secondary" href="https://publiclab.org/map/#<%= current_user.get_value_of_power_tag("zoom").to_i%>/<%=current_user.lat.to_f %>/<%=current_user.lon.to_f %>">
-      <i class="fa fa-map-marker" style="color:#c40;"></i> Nearby Activity</a>
+      <i class="fa fa-map-marker" style="color:#c40;"></i> <%= translation('dashboard_v2.sidebar.nearby_activity') %></a>
     </div>
   <%else %>
     <div style="margin: 2.2rem 0;">
-      <a class="btn btn-lg btn-outline-secondary blurred-location-input" submit_to="/profile/tags/create/<%= current_user.uid %>"><i class="fa fa-map-marker" style="color:#c40;"></i> Add your location</a>
+      <a class="btn btn-lg btn-outline-secondary blurred-location-input" submit_to="/profile/tags/create/<%= current_us
er.uid %>"><i class="fa fa-map-marker" style="color:#c40;"></i><%= translation('dashboard_v2.sidebar.add_location') %> <
/a>
       <p style="font-size: smaller; margin: 0.3rem 0;">
-        Add your location to see work near you<br>
-        <a href="https://publiclab.org/location-privacy" target="_blank">About location privacy »</a>
+      <%= translation('dashboard_v2.sidebar.location_work') %><br>
+        <a href="https://publiclab.org/location-privacy" target="_blank"><%= translation('dashboard_v2.sidebar.location
_privacy') %> »</a>
       </p>
     </div>
   <% end %>

@@ -60,7 +60,7 @@
     <% end %>
     <!-- Twitter follow button, options can be adjusted here: https://publish.twitter.com/# -->
     <div class="social-media-btn">
-      <a href="https://twitter.com/PublicLab?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-
show-count="false">Follow @PublicLab</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></sc
ript>
+      <a href="https://twitter.com/PublicLab?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-show-count="false"><%= translation('tag.blog.follow_publiclab') %></a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
     </div>
```

- [ ] 📝 **Update** the file [config/locales/en.yml](https://github.com/publiclab/plots2/blob/main/config/locales/en.yml) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

```diff
@@ -361,6 +361,7 @@ en:
         ask_question: "Ask a question"
         post_event: "Post an event"
         tell_story_blog: "Tell your story on the blog"
+        toggle_dropdown: "Toggle Dropdown"
       dashboard: "Dashboard"
       community_research: "Community research"
       welcome: "Welcome"

@@ -478,6 +479,7 @@ en:
       views: "views"
       read_more: "Read more"
       spam: "Spam"
+      follow_publiclab: "Follow @PublicLab"
     index:
       popular_tags: "Popular tags"
       browse_popular_tags: "We use tags to group research by topic -- browse below
```
- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by Sushil333**
Fixes #11074 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/like/_like.html.erb [hunk 1]
```diff
@@ -12,11 +12,11 @@
     <% if node.type == 'page' %>
       <% if node.has_tag('locked') && (current_user ? current_user.basic_user? : true) %>
         <a href="/wiki/locked" class='btn btn-outline-secondary btn-circle btn-sm' rel="tooltip" title="This page is locked. Click to learn why." data-placement="top">
-          <i class="ml-1 ff fa fa-lock"></i>
+          <i class="fa fa-lock"></i>
         </a>
       <% elsif current_user && current_user.first_time_poster %>
         <a class='btn btn-outline-secondary btn-circle btn-sm' rel="tooltip" title="You can create the wiki once your research note/question is approved by moderators. Click to learn why." data-placement="top" href="https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters">
-          <i class="ml-1 ff fa fa-lock"></i>
+          <i class="fa fa-lock"></i>
```
Comment (TildaDares) [inline app/views/like/_like.html.erb:19]: Hi @Sushil333, this changes aren't required in this PR. Thanks!

Comment (TildaDares) [inline app/views/like/_like.html.erb:19]: Hi @Sushil333, these changes aren't required in this PR. Thanks! 

Can you also remove the `package.json` and `yarn.lock` files? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/like/_like.html.erb (hunk 1 above)
Comment (Sushil333) [reply app/views/like/_like.html.erb:19]: should I delete package.json & yarn.lock?

**Comment by Sushil333** [comment]
hi, @TildaDares,
I have made the required changes. (pasted original package.json & yarn.lock) and also remove the above changes.

File: app/views/like/_like.html.erb (hunk 1 above)
Comment (TildaDares) [reply app/views/like/_like.html.erb]: Hi @Sushil333, the files are still there. You also don’t need to delete the files, just copy and paste the original into your local version of the file.

File: app/views/sidebar/_dashboard.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <div class="btn-group" role="group">
       <a class="btn btn-primary btn-lg dropdown-toggle fa-white" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
         <span class="caret"></span>
-        <span class="sr-only">Toggle Dropdown</span>
+        <span class="sr-only"><%= translation('dashboard._header.toggle_dropdown') %></span>
```
Comment (TildaDares) [inline app/views/sidebar/_dashboard.html.erb:7]: Hi @Sushil333, we need just one more change and we should be good. Thanks!
```suggestion
        <span class="sr-only"><%= translation('dashboard._header.dropdown.toggle_dropdown') %></span>
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Pull request closed without merging**